common/src/main/java/com/wynntils/wynn/event/SpellCastEvent.java

```diff
@@ -4,14 +4,20 @@
  */
 package com.wynntils.wynn.event;
 
+import com.wynntils.wynn.objects.SpellDirection;
 import com.wynntils.wynn.objects.SpellType;
-import net.minecraftforge.eventbus.api.Event;
 
-public class SpellCastEvent extends Event {
+/**
+ * Fired upon successful, completed spell cast.
+ * FIXME: This event fires upon every three-click sequence, no matter if the spell was actually casted or not.
```

Comment:
Maybe rename it TrySpellCastEvent instead then, to signal that we are not sure we managed to cast a spell. Or perhaps, arguably, we cast a spell, but we are not sure if it is *successful* or not.